test/common/wpt.js

```diff
@@ -6,9 +6,92 @@ const fs = require('fs');
 const fsPromises = fs.promises;
 const path = require('path');
 const events = require('events');
+const os = require('os');
 const { inspect } = require('util');
 const { Worker } = require('worker_threads');
 
+function getBrowserProperties() {
```

Comment:
Can you add a comment about where we can find the attributes for the outcome of this function?